pkg/apis/triggers/v1alpha1/trigger_binding_validation.go

```diff
@@ -21,10 +21,12 @@ import (
 	"knative.dev/pkg/apis"
 )
 
+// Validate TriggerBinding.
 func (t *TriggerBinding) Validate(ctx context.Context) *apis.FieldError {
 	return t.Spec.Validate(ctx)
 }
 
+// Validate TriggerBindingSpec.
```

Comment:
if you feel like it, this is a place where it would make sense to say something like "there is nothing to validate in a TriggerBindingSpec b/c xyz"

Reply:
This comment reminded me there actually is a bit of validation we can do... Added now.